ndcube/asdf/resources/schemas/global_coords-0.1.0.yaml

```
@@ -0,0 +1,30 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "asdf://sunpy.org/ndcube/schemas/global_coords-0.1.0"
+
+title:
+  Represents the ndcube GlobalCoords object
+
+description:
+  Represents the ndcube GlobalCoords object
+
+type: object
+properties:
+  internal_coords:
+    type: object
+    patternProperties:
+      .*:
+        type: array
+        items:
+            - type: string
+            - type: object
+              oneOf:
+                - tag: "tag:stsci.edu:asdf/unit/quantity-*"
+                - tag: "tag:astropy.org:astropy/coordinates/skycoord-*"
```

```suggestion
    additionalProperties:
      type: array
      items:
          - type: string
          - type: object
            oneOf:
              - tag: "tag:stsci.edu:asdf/unit/quantity-*"
              - tag: "tag:astropy.org:astropy/coordinates/skycoord-*"
```
Hopefully I didn't mess up the indenting with this suggestion. I think `additionalProperties` should work here.

---

Right, `additionalProperties` would be more appropriate here.